client/cypress/integration/query/parameter_spec.js

```diff
@@ -364,26 +375,22 @@ describe('Parameter', () => {
         .should('contain', '2');
     });
 
-    it('applies parameter changes', () => {
-      cy.getByTestId('ParameterName-test-parameter-1')
-        .find('input')
-        .type('Redash');
-
-      cy.getByTestId('ParameterName-test-parameter-2')
-        .find('input')
-        .type('Redash');
-
-      cy.location('search').should('not.contain', 'Redash');
-
-      // listen to results
-      cy.server();
-      cy.route('POST', 'api/queries/*/results').as('Results');
+    it('applies changes from "Apply Changes" button', () => {
+      expectAppliedChanges(() => {
+        cy.getByTestId('ParameterApplyButton').click();
+      });
+    });
 
-      cy.getByTestId('ParameterApplyButton')
-        .click();
+    it('applies changes from "Execute" button', () => {
+      expectAppliedChanges(() => {
+        cy.getByTestId('ExecuteButton').click();
+      });
+    });
 
-      cy.location('search').should('contain', 'Redash');
-      cy.wait('@Results');
+    it('applies changes from "meta+enter" keyboard shortcut', () => {
+      expectAppliedChanges((input) => {
+        input.type('{meta}{enter}');
```

Comment:
I'm not so sure what's the `{meta}` key on Linux :thinking:, will check that as Cypress is failing :sweat_smile: .

Reply:
For Linux (thus CI) it needs to be `{ctrl}` or `{alt}`